master/getting-started/kubernetes/upgrade.md

```diff
@@ -158,12 +141,12 @@ To upgrade the CNI config (typically located in /etc/cni/net.d) simply make the
 config file.  It will be picked up by the kubelet automatically for Kubernetes v1.4.0+.  For older versions
 of Kubernetes you must restart the kubelet for changes to be applied.
 
-#### Upgrading the Calico Policy Controller
+#### Upgrading the Calico policy controller / kube-contollers
```

Comment:
First of all, "contollers" is not a word.... (!!) LOL!

Secondly, I think it is a bit awkward to have this slash treatment here in a heading like this. I would really recommend losing "/ kube-contoller" altogether....so you could just delete this part. That would be ideal IMO